apps/condo/domains/common/components/tasks/TaskProgress.tsx

```diff
@@ -177,6 +177,10 @@ export const TaskProgressTracker: React.FC<ITaskProgressTrackerProps> = ({ task
                     handledTerminalStatesOfTasksIds.push(record.id)
                     onCancel(record)
                 }
+                if (record?.status === TASK_ERROR_STATUS && isFunction(onError)) {
```

Comment:
Please try to avoid of using optional chaining. It's better to check with lodash get or another way